### PR TITLE
button-functions: add doc about roll-pitch-toggle

### PR DESCRIPTION
**Pull request by jaxxzer**
Fix https://github.com/bluerobotics/ardusub-gitbook/issues/68

### CODE REVIEW

**Comment by Williangalvani** [comment]
what is the limit for ANGLE_MAX? 90?

**Comment by jaxxzer** [comment]
@patrickelectric fyi

**Comment by jaxxzer** [comment]
@Williangalvani Max is 80 (sometimes these can be exceeded): https://www.ardusub.com/operators-manual/full-parameter-list.html#anglemax-angle-max

File: operators-manual/button-functions.md [hunk 1]
```diff
@@ -19,6 +19,17 @@ Joystick buttons can be configured to perform different functions. Each button m
 - input_hold_toggle: Toggle holding current joystick axis inputs (cruise control on/off)
 - roll_pitch_toggle: Toggle between roll/pitch and forward/lateral control on joystick input
 
+### Roll and Pitch Control
+
+[Presently](https://github.com/ArduPilot/ardupilot/issues/9808) (due to legacy/inheritance reasons), there is no convenient way to control the roll and pitch of and ArduSub vehicle with QGroundControl. The available solution is to use the roll_pitch_toggle joystick button function. When this button is pushed, the forward and lateral joystick axes toggle between controlling the forward and lateral motion of the vehicle and controlling the roll and pitch of the vehicle.
+
+The roll and pitch input behaves differently in different flight modes. See the [flight mode documentation](/operators-manual/flight-modes) for more information about how the different flight modes operate.
+
+- MANUAL: The roll and pitch input control the **rotation rate** of the vehicle. As long as the joystick input is deflected, the vehicle will continue to rotate. A centered joystick input commands a rotation rate of zero.
```
Comment (patrickelectric) [inline operators-manual/button-functions.md]: > Note: MANUAL doesn't run a control loop, the vehicle will not hold actively any orientation change.

**Pull request merged**